lib/exhal/document.ex

```diff
@@ -69,9 +69,14 @@ defmodule ExHal.Document do
   @doc """
     **Deprecated**
 
+    See to_json_map/1
+    """
+  def to_json_hash(doc), do: to_json_map(doc)
```

Comment:
Exactly. Is it worth it? I could just remove the function and bump the major version.